### ISSUE TITLE
Should the BigInteger library provide uint division functions or methods?

### ISSUE THREAD

**Issue by lydia-duncan** [chapel-lang/chapel#17712]
Today we wrap the versions that take `bigint` arguments as methods, but GMP also defines a number of division functions on uints that aren't present in the BigInteger module.  Should the BigInteger module provide these?

I lean towards ensuring the GMP module handles it and not worrying about it here, though I can see potentially making a submodule or sibling module that provides faster uint division.

**Comment by bradcray** [comment]
I agree with the notion to leave these out of BigInteger if there's nothing particularly bigint- / big-integer-specific about them.
